velox/connectors/hive/HiveConfig.h

```diff
@@ -110,6 +110,9 @@ class HiveConfig {
   static constexpr const char* kFileColumnNamesReadAsLowerCaseSession =
       "file_column_names_read_as_lower_case";
 
+  static constexpr const char* kPartitionPathAsLowerCaseSession =
+      "partition_path_as_lower_case";
```

Comment:
Which Spark property does this correspond to?

Reply:
@mbasmanova There is no corresponding parameter configuration in Spark. In Spark, the partition path is calculated directly based on the partition column name. However, in Presto, when calculating the partition path, the partition column name needs to be converted to lowercase. This PR aims to differentiate the distinct behaviors of Spark and Presto in Velox.